Riot/Modules/Spaces/SpaceDetail/SpaceDetailViewController.swift

```diff
@@ -202,27 +207,27 @@ class SpaceDetailViewController: UIViewController {
     private func renderLoaded(parameters: SpaceDetailLoadedParameters) {
         self.activityPresenter.removeCurrentActivityIndicator(animated: true)
         
-        guard let summary = parameters.space.summary else {
-            MXLog.error("[SpaceDetailViewController] setupViews: no summary found")
-            return
-        }
-        
-        if summary.membership != .invite {
-            self.inviterPanelHeight.constant = 0
-        } else {
+        switch parameters.membership {
+        case .invite:
             self.joinButton.isHidden = true
             self.inviteActionPanel.isHidden = false
+        case .join:
+            self.inviterPanelHeight.constant = 0
+            self.joinButton.setTitle(VectorL10n.open, for: .normal)
+            self.isJoined = true
+        default:
+            self.inviterPanelHeight.constant = 0
```

Comment:
I didn't see call to `view.layoutIfNeeded` or equivalent after all constant changes. Is it intended?